pages/ShibbolethHandler.inc.php

```diff
@@ -538,9 +538,9 @@ function _registerFromShibboleth() {
 		$uin = $_SERVER[$uinHeader];
 		$userEmail = $_SERVER[$emailHeader];
 		$userFirstName = $_SERVER[$firstNameHeader];
-		$userLastName = $_SERVER[$lastNameHeader];
+		
 
-		if (empty($uin) || empty($userEmail) || empty($userFirstName) || empty($userLastName)) {
+		if (empty($uin) || empty($userEmail) || empty($userFirstName)) {
```

Comment:
Can you alter this PR to make the familyName optional, rather than removed entirely?